src/phoenix/experimental/evals/models/base.py

```diff
@@ -148,16 +148,16 @@ def _raise_import_error(
 
     @abstractmethod
     def get_tokens_from_text(self, text: str) -> List[int]:
-        raise NotImplementedError
+        ...
 
     @abstractmethod
     def get_text_from_tokens(self, tokens: List[int]) -> str:
-        raise NotImplementedError
+        ...
 
     @abstractproperty
     def max_context_size(self) -> int:
-        raise NotImplementedError
+        ...
 
     @abstractproperty
     def encoder(self) -> "Encoding":
-        raise NotImplementedError
+        ...
```

Comment:
Out of curiosity, does this change affect the behavior of the derived classes?

Reply:
This might be a matter of taste but to me it does not make much sense to `raise NotImplementedError` in both the parent and the child. The child is already forced by the parent (being an abstract class) to include those methods. So, to my taste and given the boundaries, I like leaving the parents methods unimplemented and raising in the children methods because we have not yet added implementation